src/components/Accounts/Actions/WarningModal.js

```diff
@@ -26,7 +26,7 @@ import {
 } from '../../util';
 
 import css from './modal.css';
-import { localizeCurrencyAmount } from '../../util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../util/localizeCurrencyAmount';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import